src/components/pages/Home/HomeFuseCard.tsx

```diff
@@ -39,18 +41,18 @@ const HomeFuseCard = ({
 
     const NUM = 3;
 
-    const symbols: string[] = [];
-
-    pool.assets.forEach((a, i) => {
-      const { symbol } = a.underlying;
-      if (i < NUM && symbol) symbols.push(symbol!);
```

Comment:
In what case would the symbol be `undefined`? My VSCode TypeScript Intellisense says that this can only ever be a `string`.